ui/analysis_options.yaml

```diff
@@ -1,5 +1,4 @@
 analyzer:
-  strong-mode: true
```

Comment:
this is deprecated and causes vs-code to completely flip out. not sure if it should go here or not though :thinking:

Reply:
That is true. Starting with Dart 2 there is _only_ strong mode, so no need to tell the analyzer. We'll just merge it with this, thanks.